packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml

```
@@ -103,6 +103,7 @@ $defs:
                     - stackedAreaDefault
                     - owid-distinct
                     - default
+                    - OwidDistinctLines
```

To complete all new color values I'd suggest this (OwidDistinctLines is included in the middle):
```suggestion
                    - ContinentsLines
                    - SingleColorDenim
                    - SingleColorTeal
                    - SingleColorPurple
                    - SingleColorDustyCoral
                    - SingleColorDarkCopper
                    - OwidCategoricalA
                    - OwidCategoricalB
                    - OwidCategoricalC
                    - OwidCategoricalD
                    - OwidCategoricalE
                    - OwidEnergy
                    - OwidEnergyLines
                    - OwidDistinctLines
                    - BinaryMapPaletteA
                    - BinaryMapPaletteB
                    - BinaryMapPaletteC
                    - BinaryMapPaletteD
                    - BinaryMapPaletteE
                    - SingleColorGradientDenim
                    - SingleColorGradientTeal
                    - SingleColorGradientPurple
                    - SingleColorGradientDustyCoral
                    - SingleColorGradientDarkCopper
```